.github/workflows/on-merge.yml

```diff
@@ -40,6 +40,7 @@ jobs:
       ref: ${{ github.ref }}
       version: ${{ needs.info.outputs.version }}
       enable-coverage: true
+      enable-race-detection: true
```

Comment:
Doesn't this make the binary we push to the GH release be a -race binary as well?

Reply:
No, we're rebuilding the binaries [later](https://github.com/pulumi/pulumi/blob/master/.github/workflows/ci.yml#L403) without code coverage support already, to solve https://github.com/pulumi/pulumi/issues/14799.

Reply:
How does `ci.yml` pick up this field?

Reply:
It doesn't :facepalm: thanks for catching this, I fixed it now.